src/settings/tabtexteditor.cpp

```diff
@@ -0,0 +1,155 @@
+#include "tabtexteditor.h"
+
+TabTextEditor::TabTextEditor(QWidget *parent)
+    : QWidget(parent)
+{
+    QHBoxLayout *mainLayout = new QHBoxLayout;
+    mainLayout->addStretch(1);
+
+    // getting data from QSettings
+    QSettings s;
+    styleCurrernt = (s.contains("analizerStyle") ? s.value("analizerStyle").toString() : styleDefault);
+    editorFontSizeCurrernt = (s.contains("analizerFontSize") ? s.value("analizerFontSize").toString() : editorFontSizeDefault);
+    fontCurrent = (s.contains("analizerFontName") ? s.value("analizerFontName").toString() : fontDefault);
+
+    //      STYLE
+    // set a label
+    QLabel *editorStyle = new QLabel(tr(" Editor style"));
+    pcomboEditorStyle = new QComboBox(this);
+    pcomboEditorStyle->addItems(editorStylesList);
+    mainLayout->addWidget(editorStyle);
+    mainLayout->addWidget(pcomboEditorStyle);
+
+    // set combobox acording to settings
+    pcomboEditorStyle->setCurrentIndex(0);
+    for(int i = 0; i < editorStylesList.size(); ++i)
+    {
+        auto &item = editorStylesList[i];
+        if (item == styleCurrernt)
+            pcomboEditorStyle->setCurrentIndex(i);
+    }
+    connect(pcomboEditorStyle,  SIGNAL(currentIndexChanged(const QString&)),
+                                SLOT(onChangeStyle(const QString&)));
```

Comment:
does it work? Didn't you miss the object parametr which call SLOT(onChangeStyle(const QString&)?